helios-client/src/main/java/com/spotify/helios/common/JobValidator.java

```diff
@@ -529,4 +523,34 @@ private boolean validateRepositoryName(final String repositoryName,
   private boolean legalPort(final int port) {
     return port >= 0 && port <= 65535;
   }
+
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  public static class Builder {
+
+    private boolean shouldValidateJobHash = true;
+    private boolean shouldValidateAddCapabilities = false;
```

Comment:
if you really want to make this optional (and I don't think it needs to be), it should at least default to true, so that it isn't accidentally left in the off position